dataminer/Functions/Video_Thumbnails/Configuring_Video_Thumbnails/Allowed_paths_connection_DataMiner_proxy.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: Allowed_paths_connection_DataMiner_proxy
+---
+
+# Allowed paths in case of connection via DataMiner proxy
+
+If a client does not have direct access to the video server, it can request a video feed via the DataMiner Agent acting as a proxy server.
+
+When a video feed is requested via proxy, the list of allowed URLs will be restricted for security reasons.
+
+By default, the following URLs are allowed:
+
+```txt
+http://ipaddress/axis-cgi/...
+http://ipaddress/video_thumbnails/...
+http://ipaddress/tile/...
+http://ipaddress/images/getthumbnail/...
+```
+
+If more URLs need to be added to the list of allowed URLs, do the following:
+
+1. Open `C:\Skyline DataMiner\Webpages\VideoThumbnails\Web.config`.
+
+1. Go to the *appSettings* section.
+
+1. Add all additional paths to the *ExtraAllowedPaths* key, separated by semicolons (";"):
+
+   ```xml
+   <add key="ExtraAllowedPaths" value="" />
+   ```
+
+   > [!NOTE]
+   > If you just add a single slash ("/") in the value of the ExtraAllowedPaths key, all possible URLs will be allowed.
```

Comment:
While this is correct, it should never be configured this way! This actually disables the security check, allowing a client to make any request through the proxy to the internal network, so this exposes the internal network!

Only paths should be opened here as needed, as strict as possible.